libs/data-access/database/prisma/schema.prisma

```diff
@@ -301,3 +305,87 @@ model Skill {
   @@unique([repositoryId, name])
   @@map("skills")
 }
+
+model LearningDiaryEntry {
+  id                      		String               				@id @default(uuid())
+  semesterId              		String
+  semester                		Semester            				@relation(fields: [semesterId], references: [id], onDelete: Cascade)
+  studentName			  		String
+  student						Student 							@relation(fields: [studentName], references: [username], onDelete: Cascade)
+  courseSlug               		String
+  course                  		Course              				@relation(fields: [courseSlug], references: [slug], onDelete: Cascade)
+  notes                   		String?
+  date                    		DateTime
+  start                   		DateTime
+  end                     		DateTime
+  scope							Int
+  distractionLevel        		Int?
+  effortLevel					Int?
+  learningLocationId      		String?
+  learningLocation        		LearningLocation?    				@relation(fields: [learningLocationId], references: [id], onDelete: Cascade)
+  learningGoals 		  		LearningGoal[]
+  learningTechniqueEvaluation 	LearningTechniqueEvaluation[]
+}
+
+model LearningLocation {
+	id					String 			 			@id @default(uuid())
+	name				String
+	iconURL				String
+	defaultLocation		Boolean						@default(false)
+	creatorName			String?
+	creator				Student?					@relation(fields: [creatorName], references: [username], onDelete: Cascade)
+	learningDiaryEntry 	LearningDiaryEntry[]
+}
+
+model LearningStrategie {
+  id                      			String               				@id @default(uuid())
+  name 					  			String								@unique
+  learningTechnique 				LearningTechnique[]
+}
+
+model LearningTechnique {
+	id									String 			 					@id @default(uuid())
+	name 								String
+	creatorName 						String?
+	creator								Student? 							@relation(fields: [creatorName], references: [username], onDelete: Cascade)
+	defaultTechnique					Boolean								@default(false)
+	learningStrategieId					String
+	learningStrategie					LearningStrategie    				@relation(fields: [learningStrategieId], references: [id], onDelete: NoAction)
+	learningTechniqueEvaluations 		LearningTechniqueEvaluation[]
+}
+
+model LearningTechniqueEvaluation {
+	id						String 			 			@id @default(uuid())
+	score					Int
+	learningTechniqueId		String
+	learningTechnique		LearningTechnique			@relation(fields: [learningTechniqueId], references: [id], onDelete: Cascade)
+	learningDiaryEntryId 	String
+	learningDiaryEntry		LearningDiaryEntry			@relation(fields: [learningDiaryEntryId], references: [id], onDelete: Cascade)
+}
+
+enum LearningProgeress {
```

Comment:
tippfehler "Progress"

Reply:
Ist redundant, da die Klassen von Fabian benutzt werden, heißt das fliegt eh früher oder später raus. (Nicht in dem Branch, aber später, wenn das zusammen gebaut wird)